docker-compose.yml

```diff
@@ -13,3 +13,11 @@ mailhog:
   ports:
     - "1025:1025"
     - "8025:8025"
+tomcat:
+  build: ./sigmah-docker/tomcat
+  ports:
+    - "9999:8080"
+  links:
+    - postgresql
+  volumes:
+    - ./target/sigmah-2.2-SNAPSHOT/:/usr/local/tomcat/webapps/sigmah/
```

Comment:
The documentation you put in the commit message is interesting, could you put it in a comment in this file?